app/models/lookup_keys/puppetclass_lookup_key.rb

```diff
@@ -40,4 +41,12 @@ def validate_default_value
   def puppet?
     true
   end
+
+  def check_override_selected
+    return if only_changed?(['description'])
+    return if only_changed?(['override'])
```

Comment:
@dLobatog I didn't want to allow changing other fields when changing override to false.